.ignore

```diff
@@ -1,2 +1,3 @@
 docs/prettier.min.js
```

Comment:
Can you remove this one? It's no longer used.

Also, remove docs/parser* if possible, or just docs/*.js